util/get_fisher_data.py

```diff
@@ -181,6 +193,38 @@ def get_tackle():
 
     return tackle
 
+def find_fish_by_name(fishes, name):
+    for id, value in fishes['en'].items():
+        if type(value) is list:
+            for actualName in value:
+                if actualName == name:
+                    return id
+        else:
+            if value == name:
+                return id
+
+def get_tugs(fishes):
+    tugs = {}
+    data = fish_tracker()
+    for fish in data:
+        if 'tug' in fish and fish['tug']:
+            id = find_fish_by_name(fishes, fish['name'].lower())
+            tug = None
+
+            if fish['tug'] == 'light' or fish['tug'] == 'Light':
```

Comment:
Could you store `fish['tug'].casefold()` in a local here so you don't have to compare against both upper and lowercase, and can simplify this?

Reply:
done!